packages/templates/react-urql-graphql-codegen/src/components/App.tsx

```diff
@@ -0,0 +1,34 @@
+import { useGeneratedQuery } from '../generated'
+import Fetching from './Fetching'
+import Error from './Error'
+import ListPlaceholder from './ListPlaceholder'
+import { useState } from 'react'
+
+function App() {
+  const [input, setInput] = useState("")
+  const [result] = useGeneratedQuery({
+    variables: { search: "%" + input + "%" }
+  })
+
+  const { fetching, error, data } = result
+  if (fetching) return <Fetching />;
+  if (error) return <Error error={error} />;
+
+  const handleChange = (value: string) => {
+    setInput(value)
+  }
+
+  return (
+    <>
+      <input
```

Comment:
Please, remove UI components from this template (we will have another template for material-ui ...).